recipes/impactx/meta.yaml

```diff
@@ -0,0 +1,98 @@
+{% set name = "impactx" %}
+{% set version = "22.08" %}
+{% set build = 0 %}
+{% set sha256 = "a4636fe0383f15bd4e2f128a94c537e25bdf7d09772c095ac6b2e9c66b9da385" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ECP-WarpX/impactx/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build }}
+  run_exports:
+    # releases are not (yet) compatible
+    - impactx {{ version }}
+  # wait for 22.09 for Windows builds
+  # https://github.com/ECP-WarpX/impactx/pull/205
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - ccache        # [unix]
+    - clang         # [win]
+    - cmake >=3.20.0
+    - lld           # [win]
+    # TODO: add libxml2 dep in lld-feedstock
+    - libxml2       # [win]
+    - make          # [unix]
+    - ninja         # [win]
+    - pip
+    - libgomp       # [linux]
+    - llvm-openmp   # [osx or win]
+    # ctest
+    - matplotlib-base  # [build_platform == target_platform]
+    - numpy            # [build_platform == target_platform]
+    - pandas           # [build_platform == target_platform]
+    - pytest           # [build_platform == target_platform]
+    - python           # [build_platform == target_platform]
+    - scipy            # [build_platform == target_platform]
```

Comment:
Why are these in `build` and not `host`?

Reply:
I tried in `host` and it did not find some of the packages at `ctest` time

Reply:
moved, let's see if we can get it all working :)